Marlin/Marlin_main.cpp

```diff
@@ -1720,7 +1895,8 @@ void process_commands()
 
 #ifdef ENABLE_AUTO_BED_LEVELING
     case 29: // G29 Detailed Z-Probe, probes the bed at 3 or more points.
-    	     // Override probing area by providing [F]ront [B]ack [L]eft [R]ight Grid[P]oints values
+             // Override probing area by providing [F]ront [B]ack [L]eft [R]ight Grid[P]oints values
+             // Changing GridPoints is not supported by non-linear delta printer bed leveling
         {
```

Comment:
This code has been moved into its own function, and there have been other changes to `Marlin_main.cpp`. You may get some conflicts when you go to merge the latest Development code. You may find it easiest to start with the new `Marlin_main.cpp` file and –using this diff as a guide– apply the equivalent changes in the new locations.